package-lock.json

```diff
@@ -11331,7 +11312,7 @@
     "yazl": {
       "version": "2.5.1",
       "resolved": "https://registry.npmjs.org/yazl/-/yazl-2.5.1.tgz",
-      "integrity": "sha512-phENi2PLiHnHb6QBVot+dJnaAZ0xosj7p3fWl+znIjBDlnMI2PsZCJZ306BPTFOaHf5qdDEI8x5qFrSOBN5vrw==",
+      "integrity": "sha1-o9ZdPdZZpbCTeFDoYJ8i//orXDU=",
```

Comment:
Hmm can we double check why all of these are changing?

Reply:
Was on an older version of npm. I updated it to 6.9.0, and the integrity keys have reverted AFAICT.